modules/045-snapshot-controller/docs/USAGE.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "The snapshot-controller moule: configuration examples"
+---
+
+### Using snapshots
+
+To use snapshots, you first need to create a `VolumeSnapshotClass`:
+
+```yaml
+apiVersion: snapshot.storage.k8s.io/v1beta1
+kind: VolumeSnapshotClass
+metadata:
+  name: my-first-linstor-snapshot-class
+driver: linstor.csi.linbit.com
+deletionPolicy: Delete
+```
```

Comment:
It's up to consideration. I think we can create `VolumeSnapshotClass` automatically for every supported driver

Reply:
We should do just that.